packages/frontend-language-toggle/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@govuk-one-login-frontend/frontend-language-toggle",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Reusable language toggle for GDS One Login",
   "main": "build/cjs/language-param-setter.cjs",
   "module": "build/esm/language-param-setter.js",
   "type": "module",
   "scripts": {
-    "pub": "npm run build; npm publish --access public",
+    "dev": "rollup -c --watch",
     "test": "jest",
     "test:coverage": "jest --coverage src",
-    "build": "rollup -c",
+    "build": "rollup -c && cp ./src/template.njk ./src/macro.njk ./build/",
```

Comment:
Is it possible to move this copying logic in to the rollup config? That way we know we're getting the same behaviour whether we run dev or build commands. I think this package should help: https://www.npmjs.com/package/rollup-plugin-copy